templates/base.html

```diff
@@ -67,7 +67,11 @@
     #sebunobian-huan + p + table + table,
 /* italian */
     #convertire-i-tiri-salvezza + p + table,
-    #convertire-i-tiri-salvezza + p + table + table
+    #convertire-i-tiri-salvezza + p + table + table,
+/* portuguese */
+    #convertendo-jogadas-de-protecao + p + table,
+    #convertendo-jogadas-de-protecao + p + table + table
```

Comment:
looking good. I'll double-check this when I'll try to build the HTML pages and browse the result.